spec/controllers/challenge_claims_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
+
+RSpec.describe ChallengeClaimsController, type: :controller do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { create(:user) }
+  let(:signups) { create(:challenge_signup) }
+  let(:collection) { signups.collection }
+  let(:claim) { create(:challenge_claim, collection: collection) }
+
+  describe 'index' do
+    it 'gives a notice if the collection is closed' do
+      fake_login_known_user(user)
+      allow_any_instance_of(Collection).to receive(:closed?) { true }
+      get :index, id: claim.id, collection_id: collection.name, for_user: true
+      expect(flash[:notice]).to include("This challenge is currently closed to new posts.")
+      expect(assigns(:claims))
+    end
+
+    xit 'will not allow you to see someone elses claims' do
+      second_user = create(:user)
+      fake_login_known_user(user)
+      get :index, id: claim.id, user_id: second_user.id
+      it_redirects_to_with_error(root_path, \
+                                 "You aren't allowed to see that user's claims")
+    end
+  end
+
+  describe 'show' do
+    xit 'redirects' do
```

Comment:
This seems to fail because `claim.request_prompt` is `nil` because there's no prompt. It seems to work if you make a Prompt factory:
```
# frozen_string_literal: true
require 'faker'

FactoryGirl.define do
  factory :prompt do
  end
end
```
And then change the test to:
```
    it 'redirects logged in user to the prompt' do
      request_prompt = create(:prompt, collection_id: collection.id, challenge_signup_id: signups.id)
      claim_with_prompt = create(:challenge_claim, collection: collection, request_prompt_id: request_prompt.id)
      fake_login_known_user(user)
      get :show, id: claim_with_prompt.id, collection_id: collection.name
      it_redirects_to(collection_prompt_path(collection, claim_with_prompt.request_prompt))
    end
```
(If you don't sign in first, it will redirect to the login page.)